.github/actions/tag-jira-release/action.yml

```diff
@@ -22,6 +22,10 @@ inputs:
     description: "The description of the release"
     required: false
     default: "Next awesome release"
+  is-release:
+    description: "Whether to create a release"
```

Comment:
Is this a release branch?

Reply:
Addressed in [c8a9447](https://github.com/VirdocsSoftware/github-actions/pull/13/commits/c8a9447ffce7cd6eb84401967b568aa5b47fdb7e)